From e353302e9ea2d11325749be4d0d833e725498873 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Tue, 26 Jun 2012 17:55:13 +0800
Subject: [PATCH 1697/2463] ENGR00213014-6 MXC EDID: define
 mxc_edid_parse_ext_blk function

added function mxc_edid_parse_ext_blk defined.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mxc_edid.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/mxc_edid.h b/arch/arm/plat-mxc/include/mach/mxc_edid.h
index 2b35fec..3aa1cc2 100755
--- a/arch/arm/plat-mxc/include/mach/mxc_edid.h
+++ b/arch/arm/plat-mxc/include/mach/mxc_edid.h
@@ -76,5 +76,6 @@ int mxc_edid_var_to_vic(struct fb_var_screeninfo *var);
 int mxc_edid_mode_to_vic(const struct fb_videomode *mode);
 int mxc_edid_read(struct i2c_adapter *adp, unsigned short addr,
 	unsigned char *edid, struct mxc_edid_cfg *cfg, struct fb_info *fbi);
-
+int mxc_edid_parse_ext_blk(unsigned char *edid, struct mxc_edid_cfg *cfg,
+	struct fb_monspecs *specs);
 #endif
-- 
1.7.7.4

